test/unit/storageFacade.test.ts

```diff
@@ -1,22 +1,31 @@
 import { expect, jest, test } from '@jest/globals';
 import { supabase } from '@utils/supabase';
-import { Package, Vulnerability } from '@utils/types.utils';
+import {
+  Package,
+  Vulnerability,
+  PackageViewParam,
+  VulnerabilityViewParam,
+  severityRating,
+} from '@utils/types.utils';
 import {
   writePackage,
   readPackage,
   readAllPackages,
   writeVuln,
   readVulnsBySession,
   readVulnsByPkg,
+  readVulnerabilitiesDashboard,
+  readPackagesDashboard,
 } from '@utils/storageFacade.utils';
 
 //Set up test data
+jest.setTimeout(20000);
 let sessionId = 9927;
 let packages: Package[] = [
   {
-    consRisk: null,
```

Comment:
doesnt putting it as 0 imply that we know it is 0 rather than we dont know? was it changed because we couldn't deal with a null value? even if you want to put a numerical value, would suggest -1 (or another value) so it's clear that this value hasn't been computed